.github/workflows/ci-build.yml

```diff
@@ -10,118 +10,95 @@ on:
-
-      - name: Build HTML documentation
-        run: |
-          xvfb-run tox -e doc
-      - name: Build PDF Documentation
-        run: |
-          sudo apt update
-          sudo apt-get install -y texlive-latex-extra latexmk
-          python -m pip install -r requirements/requirements_doc.txt
-          make -C doc pdf
-
-      - name: Upload HTML documentation
-        uses: actions/upload-artifact@v2.2.4
-        with:
-          name: HTML-Documentation
-          path: .tox/doc_out_html/
-          retention-days: 7
-
-      - name: Upload PDF Documentation
-        uses: actions/upload-artifact@v2.2.4
-        with:
-          name: PDF-Documentation
-          path: doc/build/latex/pyansys*.pdf
-          retention-days: 7
-
-      - name: Deploy to gh-pages
-        if: github.event_name == 'push'
-        uses: JamesIves/github-pages-deploy-action@3.7.1
-        with:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          BRANCH: gh-pages
-          FOLDER: .tox/doc_out_html/
-          CLEAN: true
-          SINGLE_COMMIT: true
-
-  release:
-    if: github.event_name == 'push' && contains(github.ref, 'refs/tags')
-    needs: [style, docs-style, docs]
+          use-python-cache: false
+          skip-install: true
+          requires-xvfb: true
+          dependencies: 'xvfb texlive-latex-extra latexmk nodejs npm graphviz'
+  
+  doc-deploy:
+    name: "Deploy developers documentation"
     runs-on: ubuntu-latest
+    needs: doc-build
     steps:
-      - uses: actions/checkout@v3
-
-      - name: Set up Python
-        uses: actions/setup-python@v1
+      - name: "Deploy the latest documentation"
+        uses: pyansys/actions/doc-deploy-dev@v1
         with:
-          python-version: ${{ env.MAIN_PYTHON_VERSION }}
+          cname: ${{ env.CNAME }}
+          token: ${{ secrets.GITHUB_TOKEN }}
 
-      - uses: actions/download-artifact@v2
-
-      - name: Display structure of downloaded files
-        run: ls -R
+  release-gitub:
+    name: "Release to GitHub"
+    runs-on: ubuntu-latest
+    if: github.event_name == 'push' && contains(github.ref, 'refs/tags')
+    needs: doc-deploy
+    steps:
 
-      - name: Release
-        uses: softprops/action-gh-release@v1
-        with:
-          generate_release_notes: true
-          files: |
-            ./*PDF*/*.pdf
+    - name: "Download HTML documentation"
+      uses: actions/download-artifact@v3
+      with:
+        name: documentation-html
+        path: documentation-html
+
+    - name: "Zip HTML documentation"
+      uses: vimtor/action-zip@v1
+      with:
+        files: documentation-html
+        dest: documentation-html.zip
+
+    - name: "Download PDF documentation"
+      uses: actions/download-artifact@v3
+      with:
+        name: documentation-pdf
+        path: documentation-pdf
+
+    - name: "Zip PDF documentation"
+      uses: vimtor/action-zip@v1
+      with:
+        files: documentation-pdf
+        dest: documentation-pdf.zip
```

Comment:
Don't really know if zipping the pdf file is really necessary. I would only zip the HTML docs (since it's composed of multiple files)- But the PDF docs are just one.

Reply:
Makes sense! Fixing this... ⚙️

Reply:
Solved in f19aee5.